recipe/meta.yaml

```diff
@@ -44,7 +44,7 @@ test:
     - nose
   commands:
     - f2py -h
-    - python runtests.py --no-build -- --nose-arg -vv
```

Comment:
but now how do we see what tests failed?

Reply:
Lets have it actually run the tests first :D

Reply:
good point. Want to save CIs by skipping non-linux?

Reply:
meh, fast-finish should deal with it mostly

Reply:
Looks like we get details on which errors run.  For py37 seems there are some tests that rely on certain behavior of warnings.  We probably need to exclude those.

Reply:
we can just skip py37.


I think the numpy macro should be something like:
```
- {{ set_compatibile_numpy('1.11') }}
```
and conda forge should just deal with it.

We can have a run exports for numpy so that we no longer need to include it in the

```
- run
```
section.